swagger.json

```diff
@@ -4665,17 +4792,44 @@
                             "description": "ID of the route that this job belongs to.",
                             "example": 55
                         },
-                        "driver_id": {
-                            "type": "integer",
-                            "format": "int64",
-                            "description":  "ID of the driver assigned to the dispatch job.",
-                            "example": 444
+                        "driver": {
+                            "type": "object",
+                            "description": "Driver object",
```

Comment:
Could this driver object reference a `#/definitions/ExternalDriver`, so it can be shared among all of the response payloads?